samples/java-webapp/index.jsp

```diff
@@ -75,28 +75,41 @@
             </div>
         </div>
     </body>
+    <script src="https://cdn.jsdelivr.net/npm/axios@0.20.0/dist/axios.min.js"></script>
     <script src="node_modules/@asgardio/oidc-js/dist/main.js"></script>
     <script>
         var serverOrigin = "https://localhost:9443";
-        var clientHost = "http://localhost:3000";
+        var clientHost = "https://localhost:9443/user-portal/";
```

Comment:
should this be myaccount?

Reply:
This should actually be empty and the user should add their client host before running this app. I have made the necessary changes and updated the read me file with the right instructions.